packages/medical-domain/src/domains/repositories/time/time.service.ts

```diff
@@ -159,3 +193,24 @@ export function twoWeeksAgo(time: string | number, timezone: string): number {
 export async function waitTimeout(timeout: number): Promise<void> {
   await new Promise(resolve => setTimeout(resolve, timeout))
 }
+
+export const isEpochBetweenBounds = (epoch: number, start: number, end: number): boolean => (
+  epoch >= start && epoch < end
+)
+
+export function diffDays(start: number, end: number): number {
+  return moment.utc(end).diff(moment.utc(start)) / MS_IN_DAY
+}
+
+export function numberOfDays(start: number, end: number, daysFilter?: WeekDaysFilter): number {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏
```suggestion
export function getNumberOfDays(start: number, end: number, daysFilter?: WeekDaysFilter): number {
```